spectacles/validators/content.py

```diff
@@ -131,8 +131,9 @@ def _get_errors_from_result(
             explore: Optional[Explore] = project.get_explore(
                 model=model_name, name=explore_name
             )
+            model: Optional[Model] = project.get_model(model_name)
```

Comment:
I think we can make this a little faster by calling `get_model` on the Project first, then calling `get_explore` on the returned Model instead of on the Project, that will cut down on extra iteration.

Reply:
Agreed. I'll make this change.